examples/benchmarking/M5-forecasting.ipynb

```diff
@@ -0,0 +1,3944 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    def load():
</pre><p>Where are you using this method? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/7113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>